src/utils/getDecoratorValue.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { get } from 'lodash';
+
+import {
+  ENTITY_KEYS,
+  FORMS_SETTINGS,
+  isFormattedMessage,
+  isTranslationId,
+} from '.';
+
+export const getDecoratorValue = value => {
+  const actions = get(FORMS_SETTINGS, [ENTITY_KEYS.MAPPING_PROFILES, 'DECORATORS', 'BOOLEAN_ACTIONS'], []);
```

Comment:
As far as this method can process values for different types of decorators, consider renaming this const to `booleanActions`